docs/release-notes/winter-2019/README.md

```diff
@@ -290,8 +292,8 @@ We have updated the recommended mappings for some features that were deprecated
 **Text Tokens**
 | Deprecated token/mixin  | Equivalent token/mixin |
 |-------------------------|-------------------------|
-| cdr-text-header-1 | cdr-text-heading-serif-1100 |
-| cdr-text-header-2 | cdr-text-heading-serif-900 |
+| cdr-text-header-1 | cdr-text-heading-serif-display-1100 |
+| cdr-text-header-2 | cdr-text-heading-serif-display-900 |
```

Comment:
These can actually just stay cdr-text-heading-serif-1100 & cdr-text-heading-serif 900, vs adding display to them.

Reply:
ah i meant to put `strong` instead of `display` here, since these were previously mapped to `display` tokens